themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -116,7 +121,8 @@ function itemQueueAjax(id, el) {
   itemStatusEls[id] = el;
   itemStatusTimer = setTimeout(runItemAjaxForQueue, itemStatusDelay);
   el.addClass('js-item-pending').removeClass('hidden');
-  el.find('.status').removeClass('hidden');
```

Comment:
I'm not sure what the intention of this change is, but I'm guessing it may be related to the fact that the unknown status message is not displaying.

Reply:
Right.. Since this interacts with the OpenURL resolution I had a bit of trouble managing it all properly. Will fix.